public/plugin.ts

```diff
@@ -27,6 +27,9 @@
 import { AppMountParameters, CoreSetup, CoreStart, Plugin, PluginInitializerContext } from "../../../src/core/public";
 import { IndexManagementPluginSetup } from ".";
 import { IndexManagementPluginStart } from ".";
+import { ActionRepository } from "./pages/VisualCreatePolicy/utils/helpers";
+
+export const actionRepoSingleton = new ActionRepository();
```

Comment:
Would it make more sense to have the import actually import a concrete instantiated class of ActionRepository instead of instantiating it here? That way all other places can just import it directly from where it exists and get the singleton instance of it.

Reply:
Yeah, I think that makes sense. Will update later.